solidity/dashboard/README.md

```diff
@@ -75,6 +75,13 @@ When you don’t want to use the local version of `@keep-network/keep-core` anym
 
 A new version of staking dApp is automatically deployed to `keep-dev` internal testnet after each `master` merge. dApp can be accessed at [https://dashboard.dev.keep.network/](https://dashboard.dev.keep.network/) and requires an initial setup in MetaMask before the first use. All the setup described below has to be done only one time. 
 
+### Ropsten testnet
+
+Change:
+In `solidity/dashboard/src/connectors/utils.js`
+- getChainId() return value to 3. As indicated, private chains ia the default, so you need to change if you use a different one (Ropsten = 3).
+- getWsUrl() return value to your Infura websocket URL (strongly recommend to get an Infura account if you don't have one), or find a websocket URL that connects to Ropsten and put it here.
```

Comment:
`getwsUrl` is best updated @ https://github.com/keep-network/keep-core/blob/master/solidity/dashboard/src/config/config.json, though you shouldn't need to update it.

Reply:
Understood! I've modified the wording. this would now be part of point 2. Marked as optional and if desired should be done in the location suggested (i.e. `solidity/dashboard/src/config/config.json`)